### ISSUE TITLE
NPM package iTowns

### ISSUE THREAD

**Issue by gchoqueux** [iTowns/itowns#122]
Make npm package for iTowns

**Comment by tbroyer** [comment]
Prerequisite: either move to commonjs (might not be enough though), or update our "bundling" policy and/or move our dependencies as devDependencies (if we bundle them in a UMD module, like we do today, then we no longer need to declare them as dependencies, and should no longer do it).

**Comment by autra** [comment]
I guess this is superseded by https://github.com/iTowns/itowns2/issues/243
